applications/asset_tracker_v2/src/main.c

```diff
@@ -119,6 +119,16 @@ static struct module_data self = {
 	.supports_shutdown = true,
 };
 
+NRF_MODEM_LIB_ON_INIT(asset_tracker_init_hook, on_modem_lib_init, NULL);
+
+/* Initialized to value different than success (0) */
+static int modem_lib_init_result = -1;
```

Comment:
I think global static variables are not allowed to be initialized (or build warning).